horizon/commons/schemas/v1/namespace.py

```diff
@@ -19,6 +19,7 @@ class NamespaceResponseV1(BaseModel):
     description: str = Field(description="Namespace description")
     changed_at: datetime = Field(description="Timestamp of last change of the namespace data")
     changed_by: Optional[str] = Field(default=None, description="Latest user who changed the namespace data")
+    owned_by: Optional[str] = Field(default=None, description="The namespace owner")
```

Comment:
```suggestion
    owned_by: str = Field(default=None, description="The namespace owner")
```

И стоит передвинуть его немного повыше, после description